mindmeld/query_factory.py

```diff
@@ -91,30 +87,49 @@ def create_query(
         raw_text = text
         char_maps = {}
 
-        # create raw, processed maps
-        if self.preprocessor:
-            processed_text = self.preprocessor.process(raw_text)
-            maps = self.preprocessor.get_char_index_map(raw_text, processed_text)
-            forward, backward = maps
-            char_maps[(TEXT_FORM_RAW, TEXT_FORM_PROCESSED)] = forward
-            char_maps[(TEXT_FORM_PROCESSED, TEXT_FORM_RAW)] = backward
+        # Step 1: Preprocessing
+        if self.text_preparation_pipeline.preprocessors:
+            processed_text = self.text_preparation_pipeline.preprocess(raw_text)
```

Comment:
so `self.preprocessor.process` is the same as `self.text_preparation_pipeline.preprocess`?

Reply:
That's correct. However, `self.text_preparation_pipeline.preprocess` can use multiple preprocessors to initially process the raw query